idd/Energy+.idd.in

```diff
@@ -31933,10 +31933,28 @@ ZoneControl:Thermostat:OperativeTemperature,
   N1,  \field Fixed Radiative Fraction
        \minimum 0.0
        \maximum< 0.9
-  A3;  \field Radiative Fraction Schedule Name
+  A3,  \field Radiative Fraction Schedule Name
        \note Schedule values of 0.0 indicate no operative temperature control
        \type object-list
        \object-list ScheduleNames
+  A4,  \field Adaptive Comfort Model Type
+       \type choice
+       \key None
+       \key AdaptiveASH55_CentralLine
```

Comment:
The use of underscore "_" in key choices is not allowed per the naming convention "Field choice names shall only use alphanumeric characters"
https://github.com/NREL/EnergyPlus/blob/develop/doc/module-developer/src/input-concepts/data-dictionary-naming-conventions.tex#L21
please remove the underscore in these key choices and adjust the get input and i/o reference accordingly